docs/index.html

```diff
@@ -88,7 +88,7 @@
     <label class="title"><a href="https://github.com/jlongster/prettier">prettier v<span class="version"></span></a></label>
     <label><input type="number" value="80" id="printWidth"></input> printWidth</label>
     <label><input type="number" value="2" id="tabWidth"></input> tabWidth</label>
-    <label><input type="checkbox" id="singleQuote"></input> singleQuote</label>
+    <label>singleQuote <select id="singleQuote"><option value="none">none</option><option value="js">js</option><option value="jsx">jsx</option><option value="all">all</option></select></label>
```

Comment:
Ah, thanks. I'm so used to things having to be transpiled/compiled these days so I didn't even consider opening the html file directly an option...